src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const ul = document.querySelector('.gallery__list');
+const largeImg = document.querySelector('.gallery__large-img');
+
+ul.addEventListener('click', (e) => {
+  const clickEvent = e;
+
+  clickEvent.preventDefault();
+
+  if (!clickEvent) {
```

Comment:
The check `if (!clickEvent)` is unnecessary because the event object `e` will always be truthy when the event listener is triggered. You can safely remove this condition.